birt-simple/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh

```

Comment:
So thoughts for later: add a parsing feature in the run file to find the latest version. duplicate the run file to get a run.sh, run.dev.sh and a run.local.sh. (I'll take notes and add this to the Trello if this solution is the one we end up going with).

Reply:
because for now making this run script is just a one or two liner

Reply:
yeah, we'll keep this part for later when we'll integrate every piece of software under 1 installer